src/api/records.tsx

```diff
@@ -194,12 +195,12 @@ export const useRecordsPaginated = (): UseQueryResult<
 export const getFormattedAxisData = (
   record: Record,
   axisName: string
-): number | Date => {
-  let formattedData: number | Date = NaN;
+): number => {
+  let formattedData = NaN;
 
   switch (axisName) {
     case 'timestamp':
-      formattedData = parseISO(record.metadata.timestamp);
+      formattedData = new Date(record.metadata.timestamp).getTime();
```

Comment:
Is it not better to keep using the `parseISO` function as that's what `useRecordsPaginated` uses? Just for consistency

Reply:
Okay so I had to look this up to double check exactly what the difference is. As I understand it, `parseISO` takes into account ISO-formatted date-time strings, which is basically what the API is giving us right now. So `parseISO` is the best function here and I'll switch that out to make use of it.

In this instance, we specifically need this date-time value as a unix timestamp, hence why I also call `getTime()` on it. But I presume you're referring specifically to the use of `new Date()` and not this part.

Reply:
Yeah I was referring to the `new Date` bit 👍